media-libs/mesa/mesa-9999.ebuild

```diff
@@ -184,6 +184,10 @@ BDEPEND="
 	${PYTHON_DEPS}
 	opencl? (
 		>=sys-devel/gcc-4.6
```

Comment:
I'm pretty sure this requirement was something to do with some C++ madness in Clover. I'm going to speculatively remove it.